### ISSUE TITLE
training loss weight for different scale depth prediction

### ISSUE THREAD

**Issue by daisily727** [FilippoAleotti/mobilePydnet#22]
the results of my network is quite blur. I feel it is about the loss.  
1. Can I know if you used multi-scale graidient loss? I saw the mega depth paper used 4 scales
2. Can I know your weight setting for the loss computation on depth prediction at different resolutions, i.e. prediction1, prediction2, and prediction3 in your model. did you used the same weight of those three?

Many thanks!

**Comment by FilippoAleotti** [comment]
Hi,
Yes, I used multi-scale gradient loss, with weights of 0.5, 0.25 and 0.125 for the full, half and quarter resolution predictions respectively. However, lower scale predictions are upsampled to fullres before the overall loss computation.
